src/AzureIntegration/src/Microsoft.AspNetCore.AzureAppServices.HostingStartup/Microsoft.AspNetCore.AzureAppServices.HostingStartup.csproj

```diff
@@ -7,6 +7,7 @@
     <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;azure;appservices</PackageTags>
+    <IsProductPackage>true</IsProductPackage>
```

Comment:
Super happy to be tracking this kind of thing in the csproj instead of some random csv.